webgui/src/flipmouse/main.js

```diff
@@ -1,9 +1,39 @@
 //according to:
 //https://electronjs.org/docs/tutorial/first-app
 
-const { app, BrowserWindow } = require('electron')
+
+let window;
+let isQuiting;
+let tray;
+
+
+const { app, BrowserWindow, Tray, Menu} = require('electron')
```

Comment:
this won't work in a browser -> maybe we have to move the electron app to a separate folder instead of integrating it into the existing web-GUI?!

Reply:
all these things could also be moved to another module `electronUtil.js` or maybe better `electonApp.js` where here only an `electronApp.init()` method is called.

Reply:
For electron, afaik you need this main.js, it is like the index.html.
Normally, when on the FLipMouse, this file won't be loaded.
Maybe we add a grunt rule, to exclude this file when minifiying & zipping for the FM3?

Reply:
ah, ok - yes, if this file is only loaded in electron, it's OK. I thought `main.js` also includes things for the web-version. Then I think it's currently also ok to not separate the electron app and web-app in two separate folders, especially if you move as much as possible in a file like `electronApp/Util.js` and therefore keep things separated.

I don't think we have to exclude `main.js` from the grunt rule, because if it's not loaded in index.html, it's also irrelevant for web-app load time. Just in case you need more space in ESP32 storage you could remove the zipped version before copying it to ESP32.